Game.cpp

```diff
@@ -57,7 +57,8 @@ Game::StartWatching(BMessenger *target)
 bool
 Game::StopWatching(BMessenger *target)
 {
-	return fTargets.erase(std::remove(fTargets.begin(), fTargets.end(), target));
+	fTargets.erase(std::remove(fTargets.begin(), fTargets.end(), target));
+	return true;
```

Comment:
The return value of this method was supposed to indicate whether anything was actually removed. Unconditionally returning `true` does not make much sense here (Incidentially, the original implementation does not make any sense either). You could either actually determine whether anything was removed or just change this method's return type to void.

While you're at it, please change the call to `erase` by adding `fTargets.end()` as a second argument so that the erase-remove idion is implemented correctly.